pages/common/jq.md

```diff
@@ -0,0 +1,23 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`jq '' {{file}}`
+
+- Read JSON stream into array and output:
```

Comment:
Nice @ostera, I like your description a lot more. Thanks for the historical context!

I also agree that `jq '' -s` is a bit more intuitive. Good call @waldyrious.